packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -202,6 +202,7 @@ export function createTestEsCluster<
     license,
     basePath,
     esArgs,
+    resources: files,
```

Comment:
FTR schema is already defines `files` property, but `resources` is what we use in serverless setup.

https://github.com/elastic/kibana/blob/d54fc701609cc9260065a35248f94a4e8e30690d/packages/kbn-es/src/utils/docker.ts#L98

For consistency using the same property.

Reply:
@Ikuni17 let me know if I use the `files` prop you introduced in the right way.

Reply:
This seems fine to me, it is serving the same purpose that was the original intention for `files`